solutions/agents/variables.tf

```diff
@@ -41,82 +41,82 @@ variable "cluster_config_endpoint_type" {
 
 variable "log_analysis_enabled" {
   type        = bool
-  description = "Deploy IBM Cloud Logging agent if set as true."
+  description = "When set to `true`, this variable deploys the IBM Cloud logging agent."
```

Comment:
When set to `true`, this variable deploys the IBM Cloud logging agent -> When set to `true`, it deploys the IBM Cloud logging agent

Reply:
@acolleen Hmm, technically this is not a module. Currently its a deployable architecture (or solution), but not sure what terminology we will use going forward, as eventually I think this will end up in the module registry as a deployable module. @SirSpidey any guidance here?

Reply:
@acolleen For consistency, I try to use `Whether` to start descriptions for booleans. As in,

> Whether to deploy the IBM Cloud logging agent. 

Which does get around the "what is this repo called?" issue.

I'm not sure whether users in the Cloud UI context could use other info in these variables, like why why not to deploy them.

Reply:
Thanks @SirSpidey +1 to your rewrite of the description which solves the issue of what to call the repo:

"Whether to deploy the IBM Cloud logging agent."